account_payment_partner/models/account_move.py

```diff
@@ -142,22 +142,7 @@ def partner_banks_to_show(self):
         self.ensure_one()
         if self.partner_bank_id:
             return self.partner_bank_id
-        if self.payment_mode_id.show_bank_account_from_journal:
-            if self.payment_mode_id.bank_account_link == "fixed":
```

Comment:
Include an override of `partner_banks_to_show` in SEPA DD module including this, indicating that the aim of this is to show the bank account where the SEPA DD is going to be charged.

Reply:
[account_banking_sepa_direct_debit](https://github.com/OCA/bank-payment/blob/14.0/account_banking_sepa_direct_debit/__manifest__.py#L14C47-L14C70) do not depend of this module.

Reply:
The real thing would have to create a glue module for that.

Anyway, this small change (<10 len if), start to cost me a lot in refactorings for features I don't need.

Reply:
Yeah, the feature was there before the refactoring. We can't lose it. I think we can keep the hack without doing a glue module, although it doesn't seem very elegant.